database/db.env

```diff
@@ -1,3 +1,3 @@
-POSTGRES_USER=splinter
-POSTGRES_PASSWORD=splinter
-POSTGRES_DB=splinter_db
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=1234
```

Comment:
Is there a way to store passwords in a more secure manner?

Reply:
@Debskij of course there are some ways but the question is if we want to introduce it now.
The simplest improvement would be to leave these variables empty and leave filling them to the person which would do the deployment. But that also means that each developer will have to do the same thing when they try to start the system from a freshly cloned repo.

I would suggest leaving credentials in configuration files until we plan the first release of the software.

Reply:
Great point, I feel that we should somehow keep the track of all the "shortcuts" that we currently use in order to simplify first local deployments. Maybe we can create a separate issue (or issues) for that case?

Reply:
@Debskij issue created https://github.com/cvlab-ai/splinter/issues/33